src/App/Entity/EventbriteData.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="eventbrite_data")
+ */
+/*final */class EventbriteData
```

Comment:
It can't (for now), because of Doctrine needing to generate, but it should be final :(